Project.toml

```
@@ -25,7 +25,7 @@ Unicode = "4ec0a83e-493e-50e2-b9ac-8f72acf5a8f5"
 
 [compat]
 CategoricalArrays = "0.10.0"
-Compat = "3.45, 4.1"
+Compat = "3.45, 4.2"
```

```suggestion
Compat = "3.46, 4.2"
```
The reason is https://github.com/JuliaLang/Compat.jl/issues/778

---

Ah, good catch! I didn't realize there were two branches of Compat being maintained.